big_tests/tests/muc_SUITE.erl

```diff
@@ -2667,7 +2668,21 @@ change_availability_status(ConfigIn) ->
     end).
 
 
-%Missing Direct Invitations (no examples)
+% Direct Invitations (example 1 from XEP-0249)
+
+direct_invite(ConfigIn) ->
```

Comment:
This test actually only checks if we don't block the invitation while it is routed, which we don't do, because it is just a one-to-one message. I think we can keep this test though, because it proves that the main use case is working with MongooseIM.

Reply:
It was when I was implementing this test that I realised this XEP is actually fully on client's side. Just like you said the only thing that server does is to route the message. I decided to keep this test for the similar reason that you're describing.